### DISCUSSION TITLE
How to add relationships in ER diagram?

### DISCUSSION THREAD

**Discussion by timcolson** [dbgate/dbgate#635] [Q&A]
I am happy that I found DbGate last week! I'm using it to explore, analyze, and document a database used by a non-profit. DbGate works great!

I'd like to make an ER digram. I can drag two tables onto a diagram, but I cannot figure out how to connect them via PK/FK relationship. 

What am I missing? Thx!

**Comment by timcolson** [comment]
After looking around more, I found the option to add a Virtual Foreign Key! 
![virtual_foreign_key](https://github.com/dbgate/dbgate/assets/1376728/aa97ca29-9e54-459a-b722-8371eea4bb3f)

The base and ref "column" labels are a little confusing. Both are set to tables, right?

**Reply by janproch** [reply]
Base column and Ref column are labels from column mapping. Try clicking "Add column", then it will be more obvious
